lib/tasks/demo.js

```diff
@@ -21,129 +25,117 @@ const defaultDemoConfig = {
 let server;
 
 function buildSass(gulp, demoConfig) {
-	return new Promise(function(resolve, reject) {
-		const src = path.join(process.cwd(), '/' + demoConfig.sass);
-		const dest = path.join(process.cwd(), '/demos/local/');
-		if (builtFiles.css.indexOf(src) === -1) {
-			if (!fs.existsSync(src)) {
-				throw new Error('Sass file not found: ' + src);
-			}
-
-			const bowerConfig = files.getBowerJson();
-			let prefixSass = '';
-			// If module has o-assets as a dependency, set local demos to use local assets
-			if (bowerConfig.dependencies && bowerConfig.dependencies['o-assets']) {
-				const moduleName = bowerConfig.name;
-				prefixSass = '@import \'o-assets/main\';\n' +
-					'@include oAssetsSetModulePaths((' + moduleName + ': ""));\n';
-			}
+	const src = path.join(process.cwd(), '/' + demoConfig.sass);
+	const dest = path.join(process.cwd(), '/demos/local/');
+	if (builtFiles.css.indexOf(src) === -1) {
+		if (!fs.existsSync(src)) {
+			throw new Error('Sass file not found: ' + src);
+		}
 
-			builtFiles.css.push(src);
-
-			const sassConfig = {
-				sass: src,
-				sassPrefix: prefixSass,
-				// For the Sass files to load correctly, they need to be in one of these two directories.
-				// Sass doesn't look in subdirectories and we can't use wildcards
-				sassIncludePaths: ['demos/src', 'demos/src/scss'],
-				sourcemaps: true,
-				buildCss: path.basename(demoConfig.sass).replace('.scss', '.css'),
-				buildFolder: dest
-			};
-
-			return build.sass(gulp, sassConfig)
-				.on('error', reject)
-				.on('end', resolve);
-		} else {
-			resolve();
+		const bowerConfig = files.getBowerJson();
+		let prefixSass = '';
+		// If module has o-assets as a dependency, set local demos to use local assets
```

Comment:
This comment (I know it hasn't changed in this refactor) could more helpfully explain why this is the case